src/components/molecules/content/blog-elements/BlogCard.svelte

```
@@ -0,0 +1,23 @@
+	interface Props {
+		image: string;
+		title: string;
+		date: string;
+		slug: string;
+	}
+
+	let { image, title, date, slug }: Props = $props();
+</script>
+
+<a
+	href={`/blog/${slug}`}
```

ESLint

[error] Found a link with a url that isn't resolved.

(svelte/no-navigation-without-resolve)